lua/nvim-treesitter/configs.lua

```diff
@@ -245,6 +265,13 @@ M.commands = {
       "-nargs=0",
     },
   },
+  TSEditQuery = {
+    run = M.edit_query_file,
+    args = {
+      "-nargs=+",
+      "-complete=custom,nvim_treesitter#available_query_groups",
+    },
+  },
```

Comment:
Maybe a version of this command to add/edit a query extension in the users config directory?

Reply:
I think the default version of the command should ask if the user wants to create a new extension if it does not already exists.

Reply:
Edit does not really create a file. So I think we should only ask for the folder.